### DISCUSSION TITLE
Provide username and password for proxy in undetected chromedriver

### DISCUSSION THREAD

**Discussion by NitinEkka** [ultrafunkamsterdam/undetected-chromedriver#1681] [Q&A]
Am trying to create a web automation with undetected chromedriver, what's the best approach to pass username and password of my proxy. I have 1 paid proxy with credentials. Kindly acknowledge at your earliest. Thank you.

**Comment by coskundeniz** [comment]
You can check `utils.py` and `proxy.py` modules in the following project.
https://github.com/coskundeniz/ad_clicker
